### PR TITLE
Secret Bootstrap: Finish processing valid secrets before failing when encountering errors

### PR DESCRIPTION
**Pull request by GeoBK**

### CODE REVIEW

**Comment by GeoBK** [comment]
cc @alvaroaleman

File: cmd/ci-secret-bootstrap/main.go [hunk 1]
```diff
@@ -417,13 +417,13 @@ func main() {
 		}
 	})
 	defer logrus.Exit(0)
-
 	ctx := context.TODO()
+	errs := []error{}
```
Comment (alvaroaleman) [inline cmd/ci-secret-bootstrap/main.go:421]: Please use `var errs []error`, there is no need to initialize this

**Comment by alvaroaleman** [comment]
/retitle Secret Bootstrap: Finish processing valid secrets before failing when encountering errors

**Pull request merged**